windows/devices/uwb/include/windows/devices/uwb/UwbCxAdapterDdiLrp.hxx

```diff
@@ -273,6 +273,18 @@ using UwbApplicationConfigurationParametersWrapper = notstd::flextype_wrapper<UW
 UwbApplicationConfigurationParametersWrapper
 From(const std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter> &uwbApplicationConfigurationParameters);
 
+using UwbSetApplicationConfigurationParametersWrapper = notstd::flextype_wrapper<UWB_SET_APP_CONFIG_PARAMS>;
+
+/**
+ * @brief Converts a vector of UwbApplicationConfigurationParameter and sessionId to UWB_SET_APP_CONFIG_PARAMS.
+ * 
+ * @param uwbApplicationConfigurationParameters
+ * @param sessionId
+ * @return UwbSetApplicationConfigurationParametersWrapper
+ */
+UwbSetApplicationConfigurationParametersWrapper
+From(const std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter> &uwbApplicationConfigurationParameters, const uint32_t sessionId);

```

Comment:
Requiring a second arg breaks the pattern we have where we can do inline round-trip conversions. We got around this for `UWB_GET_APP_CONFIG_PARAMS` by defining a wrapper structure that is a 1:1 mapping to the DDI input type (`UWB_GET_APP_CONFIG_PARAMS`). I did this locally earlier and will push the changes somewhere.

Reply:
#168